components/layouts/TheMenu/index.vue

```diff
@@ -1,12 +1,40 @@
 <template>
   <div class="menu">
     <section class="menu__mobile" tabindex="-1">
-      <Menu20 class="menu__hamburguer-link" />
-      <AppLink class="menu__link" v-bind="homeLink">
-        {{ homeLink.label }}
-      </AppLink>
-      <div class="menu__overlay" />
-      <SidebarMenu class="menu__side-menu" />
+      <div class="menu__mobile-inner-container">
+        <AppLink
```

Comment:
Not for now, but for a refactor. We should consider extracting this (the logo) to a component because we use it twice

Reply:
Follow-up in #682